tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
 envlist =
     py{27,36}-dj111{,-grp}
+    py{27,36}-pil{27,54}
```

Comment:
You'll still need to include `-djVV-` in here so it knows what django version to install.

Reply:
It may not be strictly-speaking necessary, because some of the other packages have `django` in their `install_requires`, but that means the django version will be unpredictable; better to make it deterministic.

Reply:
Ah, good note 👍Updated.